### ISSUE TITLE
Layout/SpaceAroundEqualsInParameterDefault only registered once in a line

### ISSUE THREAD

**Issue by rdunlop** [rubocop-hq/rubocop#8801]
If a single line of code has multiple offenses for the SpaceAroundEqualsInParameterDefault cop, only the first one is detected (in certain situations).

This appears to have been first reported by @jsmestad in https://github.com/rubocop-hq/rubocop/issues/8514#issuecomment-696921824

Reproduction scenario:
```ruby
class Test

  def log_msg(msg, level =:info, olevel=:error); end

end
```

Expected output:
```
Offenses:

spec/models/test.rb:3:25: C: Layout/SpaceAroundEqualsInParameterDefault: Surrounding space missing in default value assignment.
  def log_msg(msg, level =:info, olevel=:error); end
                        ^^
spec/models/test.rb:3:39: C: Layout/SpaceAroundEqualsInParameterDefault: Surrounding space missing in default value assignment.
  def log_msg(msg, level=:info, olevel=:error); end
                                      ^

1 file inspected, 2 offenses detected, 2 offenses auto-correctable
```

Actual output:
```
Offenses:

spec/models/test.rb:3:25: C: Layout/SpaceAroundEqualsInParameterDefault: Surrounding space missing in default value assignment.
  def log_msg(msg, level =:info, olevel=:error); end
                        ^^

1 file inspected, 1 offense detected, 1 offense auto-correctable
```

```
$ [bundle exec] rubocop -V
0.92.0 (using Parser 2.7.1.5, rubocop-ast 0.6.0, running on ruby 2.5.0 x86_64-linux)
```

**Comment by jsmestad** [comment]
@rdunlop thanks! 👏
